### PR TITLE
Raycaster: Display the type of camera that is not supported

### PR DESCRIPTION
**Pull request by 1993heqiang**
**Description**

Display the type of camera that is not supported.

### CODE REVIEW

File: src/core/Raycaster.js [hunk 1]
```diff
@@ -72,21 +72,23 @@ Object.assign( Raycaster.prototype, {
...
+		} else if ( camera.isOrthographicCamera ) {
 
 			this.ray.origin.set( coords.x, coords.y, ( camera.near + camera.far ) / ( camera.near - camera.far ) ).unproject( camera ); // set origin in plane of camera
 			this.ray.direction.set( 0, 0, - 1 ).transformDirection( camera.matrixWorld );
 			this.camera = camera;
 
 		} else {
 
-			console.error( 'THREE.Raycaster: Unsupported camera type.' );
+			console.error( 'THREE.Raycaster: Unsupported camera type, "' + camera.type + '".' );
```
Comment (Mugen87) [inline src/core/Raycaster.js:91]: Adding the `type` property to the `console.error()` is fine but not the refactoring above. Please undo these changes.

Comment (1993heqiang) [reply src/core/Raycaster.js:91]: For an object that is not a camera, we should not prompt "Unsupported camera type.", this is a wrong prompt.
